runtime/src/common/sgx/ias.rs

```diff
@@ -57,6 +57,10 @@ enum AVRError {
     Disabled,
     #[error("blacklisted IAS quote GID")]
     BlacklistedGID,
+    #[error("AVR did not contain TCB evaluation data number")]
+    MissingTCBEvaluationDataNumber,
+    #[error("AVR contained unacceptable TCB evaluation data number")]
+    InvalidTCBEvaluationDataNumber,
```

Comment:
Could we unify with the error from the `pcs` module:
```suggestion
    #[error("TCB evaluation data number is invalid")]
    TCBEvaluationDataNumberInvalid,
```